npm/webpack-dev-server/src/plugin.ts

```diff
@@ -28,11 +28,7 @@ export default class CypressCTOptionsPlugin implements Plugin {
     this.devserverEvents = options.devserverEvents
   }
 
-  private pluginFunc = (context: CypressCTWebpackContext, module) => {
-    if (!module.resource) {
```

Comment:
why did we need this?

Reply:
I have no idea. It's from the copy/paste Bart imported